cmd/snap-repair/runner.go

```diff
@@ -676,3 +679,35 @@ func (run *Runner) Next(brandID string) (*Repair, error) {
 		}, nil
 	}
 }
+
+// we use designated subroot keys so that we have a subtree of keys and make
+// sure not any trusted authority key can sign repairs, given that we lack
+// ATM a way to limit the purpose of keys
+var (
+	trustedRepairRootKeys []*asserts.AccountKey
+)
```

Comment:
We can drop the () while it's a single item, unless you already have follow ups that depend on this.